src/provisioner/provisioner.py

```diff
@@ -82,7 +84,13 @@ def config_apply(solution_config_url: str, cortx_conf_url: str = None,
 
         if cortx_conf_url is None:
             cortx_conf_url = CortxProvisioner._cortx_conf_url
-        cortx_conf = MappedConf(CortxProvisioner._tmp_cortx_conf_url)
+
+        # Check if consul endpoint is reachable
+        if 'consul' in cortx_conf_url:
```

Comment:
this will pass if I give yaml://tmp/consul.conf

split it from  ':' verify or use url parser

Reply:
it is a temprory code only until version 10 is launched. and current confstore path wont be changed to consul.conf. it has been hardcoded to /etc/cortx/cl;uster.conf in k8s/ do we really need split logic?

Reply:
you may leave it.